pkg/apis/resolution/v1beta1/resolution_request_types.go

```diff
@@ -64,6 +64,7 @@ type ResolutionRequestSpec struct {
 	// +optional
 	// +listType=atomic
 	Params []pipelinev1.Param `json:"params,omitempty"`
+	Url    string             `json:"url,omitempty"`
```

Comment:
If it's not otherwise marked, I'd assume this is v1beta1. Is this intended or should this be marked as alpha?

Reply:
Yes, it should be in v1beta1 I think. However, the Tekton controller will only pass it a value for this field if it is gated behind the feature flag.

Reply:
If you want breathing room to try this out first, I'd recommend starting this out as alpha. Otherwise this would immediately be in scope for beta compatibility if you wanted to make changes (e.g. 9 month deprecation window for breaking changes 😭 ).

But if you're sure you want beta, full speed ahead 🫡

Reply:
I think I'm ok with this being in Beta. I think that internally, the Resolvers also uses v1beta1 requests so making this alpha would require us reverting it all to alpha I think 🤔.

Reply:
Fixed!